documentation/asciidoc/topics/yaml_annotated/cr_backup_site.yaml

```diff
@@ -12,9 +12,9 @@ spec:
         url: openshift://api.site-a.devcluster.openshift.com:6443 <6>
         secretName: lon-token <7>
       - name: NYC
-        url: openshift://api.site-b.devcluster.openshift.com:6443
-        secretName: nyc-token
+        host: infinispan-nyc.myhost.com <8>
```

Comment:
We could have an entirely separate `yaml` example. However, I thought this might help to show that these options can actually be mixed and matched for different `sites`?

Reply:
We were chatting about keeping this content as a separate procedure to avoid confusion downstream. This asciidoc is the same source for the Data Grid Operator so we want to be sure the use cases are clear. What do you think about copying proc_configuring_sites.adoc to something like proc_manually_configuring_sites.adoc and adding all the details there?

Reply:
Ah, I was not aware of the DataGrid Operator. I can do that if you want me to. Would both of those docs still be rendered from `assembly_xsite_replication` (just with different headings)?

Reply:
No worries. To be honest you probably shouldn't need to worry about downstream. The main thing here is that these changes introduce a new use case that we should make clear in the docs. That said I don't want to create too much overhead for contributors when it comes to the docs.

I've added these changes to our release plan. If you like, we can keep docs in this PR as-is and I can add a separate docs PR to break things out and apply style edits, etc. That will probably make your life a bit easier (and also mean docs don't hold up this PR too much). 

If you would like to break things out into a new procedure, that's cool too. And, yes, in that case you'd want to include the procedure that you create in `assembly_xsite_replication`.

Reply:
Either way is fine with me. If someone else that better understands the documentation structure can do it, that's preferable to me. However, I would be willing to make at least some of the changes, if that would lead to this getting released sooner?

Reply:
OK. I can handle all the heavy lifting for the documentation on this one. Like I say, I don't want to hold things up too much or create a lot of overhead with docs. We can make things really easy and just merge docs as-is with this PR if you think all the relevant details are included. I can work with that and break content into separate user stories as we've discussed.

Reply:
I hope I'm not adding too much churn to your PR but what do you think about reverting doc changes and adding simple procedure in asciidoc or markdown or whatever you're comfortable with? 

Similar to what Ryan added in his backup/restore PR: https://github.com/infinispan/infinispan-operator/pull/596/files#diff-0dc4c153215f1f66772c25de52306c668896b46d0ef7097fb447643687dba2b1 

After your PR gets merged I can work with the source file to create a user story. If this sounds good then I think we can take the same approach for future contributions. Thanks for helping us figure this out!

Reply:
Okay, I'll write an `md` file,  since that's what I'm familiar with, under the `docs/` directory.

Reply:
@oraNod, I've added a `docs/xsite.md` file with examples for both procedures.
I'm thinking I should also go ahead and update `design.adoc` to reflect the new `CRD` definitions.